### PR TITLE
DT-802 changes and updated latest spec in ctk.

### PR DESCRIPTION
**Pull request by preetamnpr**

### CODE REVIEW

File: booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java [hunk 1]
```diff
@@ -380,17 +380,14 @@ private String lastUrlSegment(String url) {
...
-    var operation = operationParams.iterator().next();
-    if (operationParams.size() > 1
-        || !(operation.equals("cancelBooking") || operation.equals("cancelAmendment"))) {
-      return "!INVALID-VALUE!";
-    }
-    return operation;
+    var cancelJsonBody = requestPayload.body().getJsonBody();
+    return cancelJsonBody.get("bookingStatus") != null ?
+      "cancelBooking" : cancelJsonBody.get("amendedBookingStatus") != null ?
+        "cancelAmendment" : "#INVALID";
```
Comment (nt-gt) [inline booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java:390]: Please split this double `?` into multiple lines, so it is easier to read and understand it.

Comment (nt-gt) [reply booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java:390]: Also, be sure to flag it as `#INVALID` if both attributes are present at the same time.

Comment (nt-gt) [reply booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java]: FYI, The remark "Also, be sure to flag it as #INVALID if both attributes are present at the same time." remains unsolved.

Comment (preetamnpr) [reply booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java]: > FYI, The remark "Also, be sure to flag it as #INVALID if both attributes are present at the same time." remains unsolved.

@nt-gt  booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java:392 Its present here

**Review by nt-gt — APPROVED** [review]

**Pull request merged**